### ISSUE TITLE
Hi Bro, grid column class type raw or callback is not work

### ISSUE THREAD

**Issue by jokeclancool** [deniskoronets/Laravel-GridView#8]
show error this htmlspecialchars() expects parameter 1 to be string, object given 

Help please . Thx u

**Comment by deniskoronets** [comment]
stack trace? grid config? can't help unless they are provided

**Comment by hyzhakus** [comment]
Error
```
htmlspecialchars() expects parameter 1 to be string, object given (View: /var/www/laravel/sroad/vendor/woo/laravel-grid-view/resources/views/renderers/default.blade.php)/var/www/laravel/sroad/vendor/laravel/framework/src/Illuminate/Support/helpers.php#251
```
Grid config
```
@grid([
    'dataProvider' => $dataProvider,
    'columns' => [
        [
            'class' => 'callback',
            'value' => function($row) {
                return $row->first_name;
            }
        ],

    ],
])
```

**Comment by hyzhakus** [comment]
If added woo/laravel-grid-view/src/Formatters/RawFormatter.php to GridView, the error disappears

**Comment by deniskoronets** [comment]
@hyzhakus in your case, first_name property is object? seem to be from the error you've get.

**Comment by deniskoronets** [comment]
@jokeclancool , @hyzhakus this issue is resolved in 2.2, found wrong usage of sorting for callback columns.
